Tasks/SqlServerDacpacDeployment/DeployToSqlServer.ps1

```diff
@@ -41,7 +41,7 @@ $sqlPackageOnTargetMachineBlock = Get-Content $sqlDeploymentScriptPath | Out-Str
 
 $sqlPackageArguments = Get-SqlPackageCommandArguments -dacpacFile $dacpacFile -targetMethod $targetMethod -serverName $serverName -databaseName $databaseName -sqlUsername $sqlUsername -sqlPassword $sqlPassword -connectionString $connectionString -publishProfile $publishProfile -additionalArguments $additionalArguments
 
-$scriptArguments = "-sqlPackageArguments $sqlPackageArguments -sqlPassword $sqlPassword -targetConnectionString $connectionString"
```

Comment:
SqlPassword and TargetConnectionString are needed here to mask the password. Since we decided that we will rely on Secure Variables and not do any custom processing, I am not sending the password or connection string now